src/usr/local/www/services_ntpd_gps.php

```diff
@@ -449,6 +483,13 @@ function build_nmea_list() {
 	base64_decode($pconfig['initcmd'])
 ))->setHelp('Commands entered here will be sent to the GPS during initialization. Please read and understand the GPS documentation before making any changes here.');
 
+$section->addInput(new Form_Checkbox(
+	'autocorrect_initcmd',
+	null,
+	'Auto correct malformed initialization commands. (default: checked).',
```

Comment:
This box is not being checked by default, and using the set/unset method of enable/disable makes doing so difficult since you don't know if it was never set or deliberately disabled. It's much simpler to store a text value such as "enabled" and "disabled" and then check that and let unset also mean enabled.